onnxruntime/test/providers/cpu/tensor/space_depth_ops_test.cc

```diff
@@ -145,5 +145,93 @@ TEST(TensorOpTest, DepthToSpaceTest_2) {
   test.AddOutput<float>("output", {2, 3, 6, 4}, result);
   test.Run();
 }
+
+TEST(TensorOpTest, DepthToSpaceTest_3) {
+  OpTester test("DepthToSpace", 11);  // create an opset 11 model with missing default attribute
+  const int64_t blocksize = 2;
+  test.AddAttribute("blocksize", blocksize);
+
+  const int64_t N = 2, C = 12, H = 3, W = 2;
+  const std::vector<float> X = {
+      0., 1., 2., 3., 4., 5., 6., 7., 8., 9., 10.,
+      11., 12., 13., 14., 15., 16., 17., 18., 19., 20., 21.,
+      22., 23., 24., 25., 26., 27., 28., 29., 30., 31., 32.,
+      33., 34., 35., 36., 37., 38., 39., 40., 41., 42., 43.,
+      44., 45., 46., 47., 48., 49., 50., 51., 52., 53., 54.,
+      55., 56., 57., 58., 59., 60., 61., 62., 63., 64., 65.,
+      66., 67., 68., 69., 70., 71., 72., 73., 74., 75., 76.,
+      77., 78., 79., 80., 81., 82., 83., 84., 85., 86., 87.,
+      88., 89., 90., 91., 92., 93., 94., 95., 96., 97., 98.,
+      99., 100., 101., 102., 103., 104., 105., 106., 107., 108., 109.,
+      110., 111., 112., 113., 114., 115., 116., 117., 118., 119., 120.,
+      121., 122., 123., 124., 125., 126., 127., 128., 129., 130., 131.,
+      132., 133., 134., 135., 136., 137., 138., 139., 140., 141., 142.,
+      143.};
+
+  test.AddInput<float>("input", {N, C, H, W}, X);
+
+  const std::vector<float> result = {
+      0., 18., 1., 19., 36., 54., 37., 55., 2., 20., 3.,
+      21., 38., 56., 39., 57., 4., 22., 5., 23., 40., 58.,
+      41., 59., 6., 24., 7., 25., 42., 60., 43., 61., 8.,
+      26., 9., 27., 44., 62., 45., 63., 10., 28., 11., 29.,
+      46., 64., 47., 65., 12., 30., 13., 31., 48., 66., 49.,
+      67., 14., 32., 15., 33., 50., 68., 51., 69., 16., 34.,
+      17., 35., 52., 70., 53., 71., 72., 90., 73., 91., 108.,
+      126., 109., 127., 74., 92., 75., 93., 110., 128., 111., 129.,
+      76., 94., 77., 95., 112., 130., 113., 131., 78., 96., 79.,
+      97., 114., 132., 115., 133., 80., 98., 81., 99., 116., 134.,
+      117., 135., 82., 100., 83., 101., 118., 136., 119., 137., 84.,
+      102., 85., 103., 120., 138., 121., 139., 86., 104., 87., 105.,
+      122., 140., 123., 141., 88., 106., 89., 107., 124., 142., 125.,
+      143.};
+  test.AddOutput<float>("output", {2, 3, 6, 4}, result);
+  test.Run();
+}
+
+TEST(TensorOpTest, DepthToSpaceTest_4) {
+  OpTester test("DepthToSpace", 11);  // create an opset 11 model with attribute present
+  const int64_t blocksize = 2;
+  test.AddAttribute("blocksize", blocksize);
+  test.AddAttribute("mode", "DCR");
+
```

Comment:
Are there any CRD tests?

Reply:
They are covered by the backend tests. The example seems complicated enough and adding a simpler test seemed odd. That was the rationale. Let me know if you have any concerns.